packages/botonic-dx/sample-config/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "scripts": {
+    "##########  BUILD": "",
+    "build": "tsc",
+    "build-ci": "tsc -b tsconfig.tests.json",
+
+    "##########  LINT": "",
+    "lint": "npm run lint-core -- --fix && npm run prettier",
+    "lint-ci": "npm run lint-core -- -c ../.eslintrc-ci.js && npm run prettier",
```

Comment:
Should this be `.eslintrc-ci.js` instead of `../.eslintrc-ci.js`?

Reply:
Yes! And I hadn't realized because this [issue](https://github.com/mantoni/eslint_d.js/pull/169) was ignoring the -c flag.
Fixed in eslint_d 10.1.3